packages/@o3r/create/src/index.it.spec.ts

```
@@ -77,7 +77,7 @@ describe('Create new otter project command', () => {
     const appPackageJson = JSON.parse(await fs.readFile(path.join(inAppPath, 'apps', appName, 'package.json'), 'utf-8'));
     // all otter dependencies in both package.json files must be pinned:
     [
-      ...Object.entries(rootPackageJson.dependencies), ...Object.entries(rootPackageJson.devDependencies), ...Object.entries(resolutions),
+      ...Object.entries(rootPackageJson.dependencies), ...Object.entries(rootPackageJson.devDependencies), ...(resolutions ? Object.entries(resolutions) : []),
```

Here as well, I think the test should fail if resolutions are not present.